pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -159,7 +160,7 @@ public Builder withProfile(ESProfile profile) {
             return this;
         }
 
-        public Builder withFiles(List<Object> files) {
+        public Builder withFiles(List<ESFile> files) {
```

Comment:
We could use Lombok's superbuilder

Reply:
Thanks, I wasn't familiar with that, I just read about it and (if I understood correctly ) if we want to change to that then  lots of classes in DSM should also change to using that ( for both getting it to work correctly as well as keeping the structure the same across the whole code). I suggest we don't make changes in other classes right now, but I'd appreciate your thoughts on this

Reply:
I am not really fan of Lombok's builder, I prefer to do it manually since `with` prefix is more cleaner in my opinion which Lombok does not provide

Reply:
```@Builder(setterPrefix = "with")``` will do exactly what you're talking about :)
Anyway, my idea is to get rid of boilerplate code wherever it is possible.